draft-ietf-jsonpath-base.md

```diff
@@ -421,6 +421,10 @@ type errors when pondering well-formedness and validity, while
 resource depletion and related errors are comparable to 500 type
 errors.)
 
+The JSON value the JSONPath query is applied to is, by definition, a valid JSON value.
+The parsing of a JSON text into a JSON value and what happens if a JSON
+text does not represent valid JSON are not defined by this specification.
```

Comment:
perhaps “not addressed” or “not discussed”?

Reply:
I chose "not defined" because it implies invalid JSON produces undefined behaviour as far as our spec is concerned.

It wouldn't be true to say "not discussed" although we could say "not discussed further". Similarly for "not addressed". The additional paragraph discusses and addresses the case of invalid JSON, albeit very briefly.

I toyed with describing invalid JSON as being "out of scope" of the specification, but "not defined" seemed clearer.

Reply:
A agree with @glyn that "not defined" is more appropriate here.